lms/djangoapps/save_for_later/helper.py

```diff
@@ -13,6 +13,7 @@
 log = logging.getLogger(__name__)
 
 USER_SENT_EMAIL_SAVE_FOR_LATER = 'edx.bi.user.saveforlater.email.sent'
+USER_SENT_REMINDER_EMAIL_SAVE_FOR_LATER = 'edx.bi.user.saveforlater.reminder.email.sent'
```

Comment:
Pleas rename these to:
```suggestion
USER_SAVE_FOR_LATER_EMAIL_SENT = 'edx.bi.user.saveforlater.email.sent'
USER_SAVE_FOR_LATER_REMINDER_EMAIL_SENT = 'edx.bi.user.saveforlater.reminder.email.sent'
```